topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -1129,37 +1335,79 @@ Here, treatment is the primary factor that we are interested in. The sequencing
 > We recommend that you add all factors you think may affect gene expression in your experiment. It can be the sequencing type like here, but it can also be the manipulation (if different persons are involved in the library preparation), other batch effects, etc...
 {: .comment}
 
+DESeq2 requires to provide for each factor, counts of samples in each category. We will thus create small collections for each of the categories.
+
+> ### {% icon hands_on %} Hands-on: Split your collection for each of these factors
+>
+> 1. {% tool [Extract element identifiers](toolshed.g2.bx.psu.edu/repos/iuc/collection_element_identifiers/collection_element_identifiers/0.0.2) %} with the following parameters:
+>    - {% icon param-collection %} *"Dataset collection"*: `all counts`
+>
+> 2. {% tool [Search in textfiles (grep)](toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_grep_tool/1.1.1) %} with the following parameters:
+>    - {% icon param-file %} *"Select lines from"*: output of **Extract element identifiers** {% icon tool %}
+>    - *"Regular Expression"*: `untreat`
+>
+> 3. {% tool [Search in textfiles (grep)](toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_grep_tool/1.1.1) %} with the following parameters:
+>    - {% icon param-file %} *"Select lines from"*: output of **Extract element identifiers** {% icon tool %}
+>    - *"Regular Expression"*: `single`
+>
+> 4. {% tool [Filter collection](__FILTER_FROM_FILE__) %} with the following parameters:
+>    - {% icon param-collection %} *"Input Collection"*: `all counts`
+>    - *"How should the elements to remove be determined?"*: `Remove if identifiers are ABSENT from file`
+>        - {% icon param-files %} *"Filter out identifiers absent from"*: Select both datasets outputs of **Search in textfiles (grep)** {% icon tool %}.
+>
+> 5. Rename each of the 4 new collections with the four categories: `paired`, `single`, `treat`, `untreat`.
```

Comment:
It is a bit confusing. As I suggested to you, using inheritable tags (like #paired, #single, #treated, #untreat) could make the tutorial easier to read.

Reply:
I agree this is a bit heavy...

Reply:
I changed it to tags. @gallardoalba would you mind to review the new paragraph?

Reply:
Looks great!